### PR TITLE
[CDAP-13915][CDAP-13916] Fixes pipelines not showing up in profiles, and makes them clickable

### PR DESCRIPTION
**Pull request by tonybach**
JIRAs:
- https://issues.cask.co/browse/CDAP-13915
- https://issues.cask.co/browse/CDAP-13916

We already changed the UI to use the new backend format in this commit https://github.com/caskdata/cdap/pull/10427/commits/a626239277becd1475e62cd76aa397de956418db, but there was one place that we missed.

### CODE REVIEW

**Review by elfenheart — APPROVED** [review]
👍 code LGTM

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 1]
```diff
@@ -252,10 +252,12 @@ features:
         Associations:
           Header:
             created: Created
+            label: Pipelines currently using "{profile}"
             last24hrsnodehr: Last 24 hrs node hours
             last24hrsruns: Last 24 hrs runs
             name: Name
             namespace: Namespace
+            noAssociations: This profile is not associated with any schedules or triggers
```
Comment (elfenheart) [inline cdap-ui/app/cdap/text/text-en.yaml]: need to fix this message. This section is not only showing schedules and triggers

**Review by ajainarayanan — APPROVED** [review]
Code 👍 LGTM

File: cdap-ui/app/cdap/text/text-en.yaml (hunk 1 above)
Comment (bdmogal) [reply cdap-ui/app/cdap/text/text-en.yaml]: The title of the section is ``Pipelines currently using this profile``. If that's the case, I think the message should be ``No pipelines currently using this profile``, no?

**Comment by tonybach** [comment]
Demoed to @bdmogal, and has green build: https://builds.cask.co/browse/CDAP-URUT60-3. Merging.

**Pull request merged**